setup.py

```diff
@@ -30,7 +30,7 @@
 with open("README.rst") as readme:
     long_description = readme.read()
 
-pymongo_ver = ">=3.11,<4"
+pymongo_ver = ">=3.12,<4"
```

Comment:
Needed to bump this here since 3.11 doesn't have `pymongo.server_api`. One less ticket!

Reply:
Great. Can you link MOTOR-699 in the commit message?

Reply:
Will do! Thanks for pulling up the ticket.